.github/workflows/performance-tests.yml

```diff
@@ -52,164 +44,219 @@ jobs:
       - uses: stackrox/actions/infra/install-infractl@v1
       - uses: "google-github-actions/auth@v1"
         with:
-          # TODO: Add new service account
-          credentials_json: "${{ secrets.GCP_RELEASE_AUTOMATION_SA }}"
+          credentials_json: "${{ secrets.GCP_PERFORMANCE_TESTER_CI_SA }}"
       - name: "Set up Cloud SDK"
         uses: "google-github-actions/setup-gcloud@v1"
         with:
           install_components: "gke-gcloud-auth-plugin"
-      - name: Download artifacts
+
+      - name: Prepare test environment
         run: |
-          infractl artifacts "${NAME//./-}" --download-dir artifacts >> "$GITHUB_STEP_SUMMARY"
+          infractl artifacts "${NAME//./-}" --download-dir artifacts > /dev/null
```

Comment:
This command is writing URLs to the artifacts to stdout that can be accessed without authentication, therefore hiding the output.